xml/installation-installation-designate-install_designate_InfoBlox.xml

```diff
@@ -22,12 +22,10 @@
   </para>
  </note>
  <important>
-  <!-- FIXME: hardcoded version numbers, check following para for SOC 8. -
-  sknorr, 2018-01-29 -->
   <para>
-   The InfoBlox integration was enabled and validated in &kw-hos; 3.x. To use
-   InfoBlox with version [4.x / 5.x] we recommend that you review the InfoBlox
-   Deployment guide which can be found here:
+   The InfoBlox integration was enabled and validated in earlier versions. To
```

Comment:
This sentence is presumably HPE-only. (I don't know for sure but it very much sounds like it.)

In any case, I think the information in this para can be reduced to "Before you enable and use InfoBlox in \&productname;, make sure to review the InfoBlox Deployment Guide at \<link...\>. Address any further questions with your InfoBlox support contact."

That way, you don't even have to begin bothering with profiling.